packages/ketcher-core/src/application/render/restruct/reatom.ts

```diff
@@ -479,11 +480,24 @@ class ReAtom extends ReObject {
           true,
         );
       }
+      const isPreviewMode =
+        options.usageInMacromolecule === UsageInMacromolecule.MonomerPreview ||
+        options.usageInMacromolecule === UsageInMacromolecule.BondPreview ||
+        options.usageInMacromolecule ===
+          UsageInMacromolecule.MonomerConnectionsModal ||
+        (options.usageInMacromolecule === undefined && !sgroup);
+
+      const isLeavingGroupAtom =
```

Comment:
I think it's better to use Atom.isSuperatomLeavingGroupAtom for this purpose, to have only one point of truth for this logic